camel-idea-plugin/src/main/java/com/github/cameltooling/idea/service/CamelService.java

```diff
@@ -71,7 +71,8 @@
  */
 public class CamelService implements Disposable {
 
-    private static final NotificationGroup CAMEL_NOTIFICATION_GROUP = NotificationGroupManager.getInstance().getNotificationGroup("Apache Camel");
+    private static final NotificationGroup CAMEL_NOTIFICATION_GROUP = NotificationGroupManager.getInstance()
```

Comment:
Please remove your commit "Introduce Explaining Variable" as it doesn't bring anything but reformatting